### DISCUSSION TITLE
HTTPS message after installation

### DISCUSSION THREAD

**Discussion by nantoSur** [hnimminh/libresbc#176] [General]
after the installation process is complete, only a message like this appears.

{
  "detail": "Not Found"
}

![image](https://github.com/user-attachments/assets/e1fdfdeb-37f7-43ad-ba67-a649be72eaa2)

**Comment by hnimminh** [comment]
Once LibreSBC had been installed successfully, you can access to online Rest API via:
https://<your-libresbc-ip>:8443/apidocs 

for more information you can refer to https://github.com/hnimminh/libresbc/wiki/Configuration 

If you're looking for UI, this may help https://github.com/hnimminh/libresbc/discussions/120
